libsrc/telestrat/initcwd.s

```diff
@@ -0,0 +1,28 @@
+;
+; Oliver Schmidt, 18.04.2005
```

Comment:
If you don't want to update this section of the file then please remove it altogether. I'm pretty sure I didn't write `BRK_TELEMON XVARS` ;-)